packages/lumx-react/src/components/link/Link.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+import { COMPONENT_PREFIX } from '@lumx/react/constants';
+import classNames from 'classnames';
+
+import { Color, ColorVariant } from '@lumx/react';
+import { IGenericProps, getRootClassName, handleBasicClasses } from '@lumx/react/utils';
+
+/////////////////////////////
+
+/**
+ * Defines the props of the component.
+ */
+
+interface ILinkProps extends IGenericProps {
```

Comment:
Since this component is really a wrapper around `<a>`, I think it would be nice to make the props extends the native anchor props.
```suggestion
interface ILinkProps extends HTMLAttributes<HTMLAnchorElement> {
```
The IGenericProps would not be necessary I think.